ambry-mysql/src/main/java/com/github/ambry/mysql/MySqlDataAccessor.java

```diff
@@ -271,10 +271,13 @@ public static boolean isCredentialError(SQLException e) {
   public void onException(SQLException e, OperationType operationType) {
     if (e instanceof SQLTransientConnectionException) {
       if (operationType == OperationType.Write) {
+        metrics.writeRate.mark();
```

Comment:
I am thinking if we can move this to caller method and use separate metrics for AccountService and AccountStatsReport. Thus, we have a clear view that how may QPS from AccountService, how many from Account Stats/ Quota Service.

Reply:
the metrics takes the store class as part of the metric name, so it's already separated.